gtfs/spec/en/reference.md

```diff
@@ -128,7 +130,7 @@ File: **Required**
 |  `agency_name` | Text | **Required** | Full name of the transit agency. |
 |  `agency_url` | URL | **Required** | URL of the transit agency. |
 |  `agency_timezone` | Timezone | **Required** | Timezone where the transit agency is located. If multiple agencies are specified in the dataset, each must have the same `agency_timezone`. |
-|  `agency_lang` | Language code | Optional | Primary language used by this transit agency. This field helps GTFS consumers choose capitalization rules and other language-specific settings for the dataset. |
+|  `agency_lang` | Language code | Optional | Primary language used by this transit agency. Should be provided to help GTFS consumers choose capitalization rules and other language-specific settings for the dataset. |
 |  `agency_phone` | Phone number | Optional | A voice telephone number for the specified agency. This field is a string value that presents the telephone number as typical for the agency's service area. It can and should contain punctuation marks to group the digits of the number. Dialable text (for example, TriMet's "503-238-RIDE") is permitted, but the field must not contain any other descriptive text. |
```

Comment:
```suggestion
|  `agency_phone` | Phone number | Optional | A voice telephone number for the specified agency. This field is a string value that presents the telephone number as typical for the agency's service area. It may and should contain punctuation marks to group the digits of the number. Dialable text (for example, TriMet's "503-238-RIDE") is permitted, but the field must not contain any other descriptive text. |
```

Reply:
Should we pick either "may" or "should" here? Does "should" imply "may"?

Reply:
"Should" and "may" have different purposes as defined by [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119). "Should" denotes a recommendation (_recommendation_ for the GTFS data producer to do something) and "may" denotes that the item is optional and identifiable as OK by a validator (phone numbers containing punctuation marks are _permitted_). This sentence has both, but they have different purposes. IMHO we should define the spec first by its behavior, so if we were to pick one I would keep "may". But both are ok to have.